### ISSUE TITLE
Add option for filtering single-tile products

### ISSUE THREAD

**Issue by j08lue** [ibamacsr/sentinelsat#84]
It might be worthwile to add an option to select only single-tile products - or exclude them - in queries.

This is related to our query tests currently failing as mentioned in #81.

**Comment by Fernerkundung** [comment]
The tests shouldn't be failing since the scihub behaviour is pretty straightforward. All products before 2016-12-06 are multiple tiles per dataset and all products afterward are single tiled. No word is out yet on whether ESA will reprocess old products to conform to the new PDS.

see also the [related news](https://scihub.copernicus.eu/news/News00124)

**Comment by Fernerkundung** [comment]
@j08lue Since you found another reason for the bug in #81 is this issue still relevant or can it be closed?

**Comment by j08lue** [comment]
@Fernerkundung Well, what I meant with this issue was also intended for general queries: I thought SciHub would distribute both the single-tile and multi-tile formats, but this, apparently is wrong.

I *do* follow the SciHub news and did read the various docs, but find both rather confusing, especially on the distribution formats...

Tl;dr yes, I am closing this.

**Comment by Fernerkundung** [comment]
> but find both rather confusing, especially on the distribution formats

Absolutely, especially since the vote is still out if the old products will be re-processerd to conform to the new format. So far we are dealing with, I think, 4 different product format iterations beeing distributed.
